src/porepy/numerics/ad/operators.py

```diff
@@ -1435,9 +1431,17 @@ def __repr__(self) -> str:
 class Tree:
     """Simple implementation of a Tree class. Used to represent combinations of
     Ad operators.
+
+    References:
+        https://stackoverflow.com/questions/2358045/how-can-i-implement-a-tree-in-python
+
+
+    Parameters:
+        operation: see :data:`Operation`
```

Comment:
```suggestion
        operation: See :data:`Operation`.
```

Reply:
You need to decide at this case if you want the documentation for this class included.
Right now it is an internal class, i.e. not present in __all__ and Sphinx realizes that and ignores it (by default).

Reply:
I'm afraid I don't understand what this comment refers to.

Reply:
I mean the class Tree, sorry.
You need to decide to make it available via __all__, otherwise Sphinx will ignore it.